src/Microsoft.Health.Fhir.Core/Features/Schema/LatestSchemaVersionHandler.cs

```diff
@@ -0,0 +1,45 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using MediatR;
+using Microsoft.Health.Fhir.Core.Exceptions;
+using Microsoft.Health.Fhir.Core.Features.Operations;
+using Microsoft.Health.Fhir.Core.Features.Security;
+using Microsoft.Health.Fhir.Core.Features.Security.Authorization;
+using Microsoft.Health.Fhir.Core.Messages.Schema;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Schema
+{
+    internal class LatestSchemaVersionHandler : IRequestHandler<GetCompatibilityVersionRequest, GetCompatibilityVersionResponse>
+    {
+        private readonly IFhirOperationDataStore _fhirOperationDataStore;
+        private readonly IFhirAuthorizationService _fhirAuthorizationService;
+
+        public LatestSchemaVersionHandler(IFhirOperationDataStore fhirOperationDataStore, IFhirAuthorizationService fhirAuthorizationService)
+        {
+            EnsureArg.IsNotNull(fhirOperationDataStore, nameof(fhirOperationDataStore));
+            EnsureArg.IsNotNull(fhirAuthorizationService, nameof(fhirAuthorizationService));
+            _fhirOperationDataStore = fhirOperationDataStore;
+            _fhirAuthorizationService = fhirAuthorizationService;
+        }
+
+        public async Task<GetCompatibilityVersionResponse> Handle(GetCompatibilityVersionRequest request, CancellationToken cancellationToken)
+        {
+            EnsureArg.IsNotNull(request, nameof(request));
+
+            if (await _fhirAuthorizationService.CheckAccess(DataActions.SchemaMigration) != DataActions.SchemaMigration)
```

Comment:
Due to some reason the api is throwing Authorization exception. I have added new enum SchemaMigration in DataAccess. Not sure what am I missing here :( Please suggest.

Reply:
@johnstairs any guidance here?
I think the controller action has `AllowAnonymous` so it will probably not be able to check roles/access for unauth'd users

Reply:
I discussed with John this morning, since schema migration will be run by admin by hitting few commands on tool, we might not need authorization check for schema migration specific apis. As of now i have removed this check. But I am all ears for any suggestions.

Reply:
I agree that the AuthN/AuthZ isn't needed for these APIs.